less/mixins/_box-shadow.less

```diff
@@ -3,22 +3,20 @@
 //    box-shadow: $shadow;
 //  }
 //}
-#box-shadow(@shadow...) when (@enable-shadows) {
+#box-shadow(@shadows...) when (@enable-shadows) {
 	// LESS PORT: In order to output the shadows correctly we have to iterate over the list and use
 	// Less’s merge feature. Without this, shadows will be output space-separated instead of comma-
 	// separated. Also, since a single shadow can be misinterpreted as multiple shadows (since it
 	// will have a length > 1) we have to include a check for the length of the first item in the
 	// list. If the length is greater than 1, then we have a list of separate shadows. If the the
 	// length is 1, then we’re looking at the first value of a single shadow, so we output `@shadow`
 	// as-is.
-	& when (length(@shadow) > 0) and (length(extract(@shadow, 1)) = 1) {
-		box-shadow: @shadow;
+	& when (length(@shadows) > 0) and (length(extract(@shadows, 1)) = 1) {
```

Comment:
Could you raise this as an issue on Less? Seems like there should be a function to normalize lists, like a `list(@shadows, ',')` function? Or maybe you should add that as a plugin? Maybe it's not needed in Less core.

Reply:
Hmm... do you mean, when parsing, Less doesn't specify how the list was created? If so, and it _can't_ be addressed via plugin, then it seems more like an issue to address in Less core, at least to give more information about those nodes.